src/components/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const mother = motherName ? people.find(p => p.name === motherName) : null;
+
+  const father = fatherName ? people.find(p => p.name === fatherName) : null;
```

Comment:
Consider optimizing the search for `mother` and `father` by using a more efficient lookup method. The current implementation uses a linear search, which can be inefficient if the `people` array is large. A map or an object keyed by names could provide constant-time lookups.